### PR TITLE
[MM-19122] Migrate tests from "model/user_access_token_test.go" to use testify

### PR DESCRIPTION
**Pull request by karanrn**
#### Summary
Modify the calls to t.Fatal and the condition checks that lead to them, replacing them with calls to the testify require or assert

#### Ticket Link
Fixes: https://github.com/mattermost/mattermost-server/issues/12545
JIRA: https://mattermost.atlassian.net/browse/MM-19122

### CODE REVIEW

**Comment by karanrn** [comment]
@mgdelacroix Can you please let me know the reasons for build failure?
I see most of the errors logged as: "An error occurred while configuring SAML Service Provider","error":"failed to load idp public certificate:"

File: model/user_access_token_test.go [hunk 1]
```diff
@@ -16,43 +18,37 @@ func TestUserAccessTokenJson(t *testing.T) {
...
 }
 
 func TestUserAccessTokenIsValid(t *testing.T) {
 	ad := UserAccessToken{}
 
-	if err := ad.IsValid(); err == nil || err.Id != "model.user_access_token.is_valid.id.app_error" {
-		t.Fatal(err)
-	}
+	err := ad.IsValid()
+	errCheck := err == nil || err.Id != "model.user_access_token.is_valid.id.app_error"
```
Comment (reflog) [inline model/user_access_token_test.go]: no need for errCheck intermediary variable. just pass the condition to `require.False`

Comment (karanrn) [reply model/user_access_token_test.go]: I have made the changes.

**Comment by stevemudie** [comment]
LGTM

**Pull request merged**